modules/flags/FeatureFlags.js

```diff
@@ -12,23 +12,29 @@ class FeatureFlags {
      * Configures the module.
      *
      * @param {boolean} flags.runInLiteMode - Enables lite mode for testing to disable media decoding.
-     * @param {boolean} flags.sourceNameSignaling - Enables source names in the signaling.
      * @param {boolean} flags.receiveMultipleVideoStreams - Signal support for receiving multiple video streams.
+     * @param {boolean} flags.sendMultipleVideoStreams - Signal support for sending multiple video streams.
+     * @param {boolean} flags.sourceNameSignaling - Enables source names in the signaling.
+     * @param {boolean} flags.ssrcRewritingEnabled - Use SSRC rewriting. Requires sourceNameSignaling to be enabled.
+     * @param {boolean} flags.enableUnifiedOnChrome - Use unified plan signaling on chrome browsers.
      */
     init(flags) {
-        this._receiveMultipleVideoStreams = flags.receiveMultipleVideoStreams ?? true;
         this._runInLiteMode = Boolean(flags.runInLiteMode);
+        this._receiveMultipleVideoStreams = flags.receiveMultipleVideoStreams ?? true;
         this._sendMultipleVideoStreams = flags.sendMultipleVideoStreams ?? true;
         this._sourceNameSignaling = flags.sourceNameSignaling ?? true;
-        this._ssrcRewriting = Boolean(flags.ssrcRewritingEnabled);
+        this._ssrcRewriting = this._sourceNameSignaling && Boolean(flags.ssrcRewritingEnabled);
 
         // For Chromium, check if Unified plan is enabled.
         this._usesUnifiedPlan = browser.supportsUnifiedPlan()
             && (!browser.isChromiumBased() || (flags.enableUnifiedOnChrome ?? true));
 
-        logger.info(`Source name signaling: ${this._sourceNameSignaling},`
+        logger.info(`Lite mode: ${this._runInLiteMode},`
```

Comment:
We don't need to log lite mode since it is used only for testing purposes.

Reply:
ok.